ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -570,6 +570,53 @@ def Conv3D(
     )
 
 
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    strides,
+    padding,
+    use_cudnn_on_gpu=False,
```

Comment:
I'm not sure why you removed this, but you should add `explicit_paddings=[],` here. The frontend function needs to have the exact same arguments as the original function.